src/containers/monitor.jsx

```diff
@@ -0,0 +1,32 @@
+const bindAll = require('lodash.bindall');
+const React = require('react');
+
+const MonitorComponent = require('../components/monitor/monitor.jsx');
+
+class Monitor extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleDragEnd'
+        ]);
+    }
+    handleDragEnd (_, {x, y}) {
```

Comment:
yeah `_` is the actual mouse event, the second param is the drag event as provided by react-draggable (with properly offset coordinates from parent, for example). I was just avoiding the `unused-var` linter error, but yeah i can name it.